src/demo/models/guide.js

```diff
@@ -80,7 +80,7 @@ const guides = [
   },
   {
     id: 'fishvtrash-training-pause4',
-    text: `A.I. learns from your choices.  If you make “wrong” choices, A.I. will learn to repeat your mistakes.`,
+    text: `A.I. learns from your choices.  If you make “wrong” choices, A.I. will learn to repeat your mistakes. Keep training.`,
```

Comment:
super nit: double space after period to be consistent?

Reply:
whoops, thanks for pointing that out.